py/desisim/templates.py

```diff
@@ -340,10 +341,9 @@ def __init__(self, objtype='ELG', minwave=3600.0, maxwave=10000.0, cdelt=2.0,
             corresponding to BASEFLUX (Angstrom).
           basemeta (astropy.Table): Table of meta-data [nbase] for each base template.
           pixbound (numpy.ndarray): Pixel boundaries of BASEWAVE (Angstrom).
-          decamwise (speclite.filters instance): DECam2014-* and WISE2010-* FilterSequence
-          gfilt (speclite.filters instance): DECam2014 g-band FilterSequence
           rfilt (speclite.filters instance): DECam2014 r-band FilterSequence
-          zfilt (speclite.filters instance): DECam2014 z-band FilterSequence
+          normfilt (speclite.filters instance): FilterSequence of self.normfilter
+          decamwise (speclite.filters instance): DECam2014-* and WISE2010-* FilterSequence
```

Comment:
In the documentation, ```rfilt```, ```normfilt```, and ```decamwise``` are all attributes, not arguments of the Class, so I think I'm in the clear.  And although the naming isn't great, the documentation is also clear that ```normfilter``` is a *string* argument (for the desired normalization bandpasses), whereas ```normfilt``` is a FilterSequence  instance and an attribute of the Class.